### PR TITLE
Update use-dkim-to-validate-outbound-email.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to enable DKIm and copy the CNAME as per the new navigation

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7448

**Comment by msbemba** [comment]
issue #https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7459 is also covered as it is the same document.

File: microsoft-365/security/office-365-security/use-dkim-to-validate-outbound-email.md [hunk 1]
```diff
@@ -87,14 +87,12 @@ Step 1: Click on the domain you wish to configure DKIM on DKIM page (https://sec
 
 ![DKIM page in the Microsoft 365 Defender portal with a domain selected.](../../media/126996261-2d331ec1-fc83-4a9d-a014-bd7e1854eb07.png)
 
-Step 2: Click on Create DKIM keys.
+Step 2: Slide the toggle to enable. You will see a pop up window stating that you need to add CNAME records
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/use-dkim-to-validate-outbound-email.md]:
```suggestion
Step 2: Slide the toggle to **Enable**. You will see a pop-up window stating that you need to add CNAME records.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MSFTTracyP Please merge this PR. Thanks

**Comment by msbemba** [comment]
Issue#[7567](https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7567) is also covered in this PR

**Comment by yogkumgit** [comment]
@MSFTTracyP Gentle reminder. Thanks

**Pull request merged**